php/README.md

```diff
@@ -95,3 +95,53 @@ Known Issues
 * HHVM not tested.
 * C extension not tested on windows, mac, php 7.0.
 * Message name cannot be Empty.
+
+## Development
+
+### Docker Image
+
+We provide a docker image for php development, which is also used in our automatic tests:
+```
+docker run --security-opt seccomp=unconfined -it protobuftesting/php_8dbe419c6df1a8b3af0ae3a267c112efb436b45c
+```
+
+Insider the docker, you need to manually do some setup:
+```
+apt-get install vim
+export VERSION=5.6 # The php runtime version
+                   # We provide 5.5, 5.5-zts, 5.6, 5.6-zts, 7.0, 7.0-zts, 7.1, 7.1-zts, 7.2, 7.2-zts, 7.3 and 7.3-zts
+                   # ls /usr/local for more details
+export CPLUS_INCLUDE_PATH=/usr/local/php-${VERSION}/include/php/main:/usr/local/php-${VERSION}/include/php/:$CPLUS_INCLUDE_PATH
+export C_INCLUDE_PATH=/usr/local/php-${VERSION}/include/php/main:/usr/local/php-${VERSION}/include/php/:$C_INCLUDE_PATH
+```
+
+### Test Native PHP
+
+```
+# Download protobuf
+git clone https://github.com/protocolbuffers/protobuf.git
+cd protobuf
+
+# Build protoc
+./autogen.sh
+./configure
+make -4
+
+# Test native php
+cd php
+composer install
+composer test
+```
+
+### Test C Exteinsion
+
+After you have finished testing the native php, you can test the c exteniosn:
+```
+cd test
+./test.sh
```

Comment:
This gives me an error:

```
/home/protobuf/php/ext/google/protobuf /home/protobuf/php/tests 
make: *** No rule to make target 'clean'.  Stop.                      
./compile_extension.sh: line 9: phpize: command not found
```

There is some other output after but it doesn't seem to be running the tests successfully.

Reply:
It looks like I'm supposed to pass an argument, like:

```
$ ./test 5.6
```